openslides/poll/access_permissions.py

```diff
@@ -36,13 +36,11 @@ class BasePollAccessPermissions(BaseAccessPermissions):
                     del poll["votesvalid"]
                     del poll["votesinvalid"]
                     del poll["votescast"]
+                    # add user_has_voted instead of voted_id array so user can check if he has voted yet
+                    poll["user_has_voted"] = user_id in poll["voted_id"]
```

Comment:
One thing here: managers do not get this field. You need to modify all polls before the permission check above.